### ISSUE TITLE
🧪 WCAG-audit

### ISSUE THREAD

**Issue by roxyfokker** [Rox-Codelabs/look-and-feel-corporate-identity#33]
#Lighthouse accessibility score
---

<img width="872" height="261" alt="Image" src="https://github.com/user-attachments/assets/2cebb0a4-2990-41e7-8414-547daae5e9f4" />

## $\color{#ff0400}{FAILED}$
---

* Background and foreground colors do not have a sufficient $\color{#ff0400}{contrast}$ ratio.
> De achtergrondkleur van de footer is zwart en de H4- en H5-teksten zijn donkerblauw. Dit kleurcontrast voldoet niet aan de WCAG-richtlijnen voor leesbaarheid, waardoor de tekst voor gebruikers met visuele beperkingen moeilijk te lezen is. Dit kan de toegankelijkheid van de website verminderen.

* $\color{#ff0400}{Heading}$ elements are not in a sequentially-descending order
> In de article cards ontbreken H3-elementen, waardoor de heading-structuur niet correct sequentieel afloopt (bijvoorbeeld H2 → H4/H5). Dit kan problemen veroorzaken voor gebruikers die screenreaders gebruiken, omdat de inhoud niet logisch wordt voorgelezen.

## $\color{#00ff00}{Passed}$
---

* `[aria-*]` attributes match their roles
> Each `ARIA` role supports a specific subset of `aria-*` attributes. Mismatching these invalidates the `aria-*` attributes.

* `[aria-hidden="true"]` is not present on the document `<body>`
> Assistive technologies, like screen readers, work inconsistently when `aria-hidden="true"` is set on the document `<body>`.

* `[aria-*]` attributes have valid values
> Assistive technologies, like screen readers, can't interpret `ARIA` attributes with invalid values.

* `[aria-*]` attributes are valid and not misspelled
> Assistive technologies, like screen readers, can't interpret `ARIA` attributes with invalid names.

* Buttons have an accessible name
> When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

* Image elements have `[alt]` attributes
> Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute.

* `[user-scalable="no"]` is not used in the `<meta name="viewport">` element and the `[maximum-scale]` attribute is not less than 5.
> Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page.

* ARIA attributes are used as specified for the element's role
> Some ARIA attributes are only allowed on an element under certain conditions.

* Elements use only permitted ARIA attributes
> Using ARIA attributes in roles where they are prohibited can mean that important information is not communicated to users of assistive technologies.

* Document has a `<title>` element
> The title gives screen reader users an overview of the page, and search engine users rely on it heavily to determine if a page is relevant to their search.

* `<html>` element has a `[lang]` attribute
> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

* `<html>` element has a valid value for its `[lang]` attribute
> Specifying a valid [BCP 47 language](https://www.w3.org/International/questions/qa-choosing-language-tags#question) helps screen readers announce text properly.

* Form elements have associated labels
> Labels ensure that form controls are announced properly by assistive technologies, like screen readers.

* Links have a discernible name
> Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.

* Lists contain only `<li>` elements and script supporting elements (`<script>` and `<template>`).
> Screen readers have a specific way of announcing lists. Ensuring proper list structure aids screen reader output.

* List items (`<li>`) are contained within `<ul>`, `<ol>` or `<menu>` parent elements
> Screen readers require list items (`<li>`) to be contained within a parent `<ul>`, `<ol>` or `<menu>` to be announced properly.

* Touch targets have sufficient size and spacing.
> Touch targets with sufficient size and spacing help users who may have difficulty targeting small controls to activate the targets.

* Uses ARIA roles only on compatible elements
> Many HTML elements can only be assigned certain ARIA roles. Using ARIA roles where they are not allowed can interfere with the accessibility of the web page.

* Image elements do not have `[alt]` attributes that are redundant text.
> Informative elements should aim for short, descriptive alternative text. Alternative text that is exactly the same as the text adjacent to the link or image is potentially confusing for screen reader users, because the text will be read twice.

**Comment by roxyfokker** [comment]
## Handmatige test
***
$\color{#00ff00}{PASS}$ of $\color{#ff0400}{FAIL}$
***

* $\color{#00ff00}{PASS}$ - Interactive controls are keyboard focusable 
> Custom interactive controls are keyboard focusable and display a focus indicator. [Learn how to make custom controls focusable](https://developer.chrome.com/docs/lighthouse/accessibility/focusable-controls/?utm_source=lighthouse&utm_medium=devtools).

* Interactive elements indicate their purpose and state
> Interactive elements, such as links and buttons, should indicate their state and be distinguishable from non-interactive elements. [Learn how to decorate interactive elements with affordance hints](https://developer.chrome.com/docs/lighthouse/accessibility/interactive-element-affordance/?utm_source=lighthouse&utm_medium=devtools). 

* The page has a logical tab order
> Tabbing through the page follows the visual layout. Users cannot focus elements that are offscreen. [Learn more about logical tab ordering](https://developer.chrome.com/docs/lighthouse/accessibility/logical-tab-order/?utm_source=lighthouse&utm_medium=devtools).

* Visual order on the page follows DOM order
> DOM order matches the visual order, improving navigation for assistive technology. [Learn more about DOM and visual ordering](https://developer.chrome.com/docs/lighthouse/accessibility/visual-order-follows-dom/?utm_source=lighthouse&utm_medium=devtools).

* User focus is not accidentally trapped in a region
> A user can tab into and out of any control or region without accidentally trapping their focus. [Learn how to avoid focus traps](https://developer.chrome.com/docs/lighthouse/accessibility/focus-traps/?utm_source=lighthouse&utm_medium=devtools).

* The user's focus is directed to new content added to the page
> If new content, such as a dialog, is added to the page, the user's focus is directed to it. [Learn how to direct focus to new content](https://developer.chrome.com/docs/lighthouse/accessibility/managed-focus/?utm_source=lighthouse&utm_medium=devtools).

* HTML5 landmark elements are used to improve navigation
> Landmark elements (<main>, <nav>, etc.) are used to improve the keyboard navigation of the page for assistive technology. [Learn more about landmark elements](https://developer.chrome.com/docs/lighthouse/accessibility/use-landmarks/?utm_source=lighthouse&utm_medium=devtools).

* Offscreen content is hidden from assistive technology
> Offscreen content is hidden with display: none or aria-hidden=true. [Learn how to properly hide offscreen content](https://developer.chrome.com/docs/lighthouse/accessibility/offscreen-content-hidden/?utm_source=lighthouse&utm_medium=devtools).

* Custom controls have associated labels
> Custom interactive controls have associated labels, provided by aria-label or aria-labelledby. [Learn more about custom controls and labels](https://developer.chrome.com/docs/lighthouse/accessibility/custom-controls-labels/?utm_source=lighthouse&utm_medium=devtools).

* Custom controls have ARIA roles
> These items address areas which an automated testing tool cannot cover. Learn more in our guide on [conducting an accessibility review](https://web.dev/articles/how-to-review?utm_source=lighthouse&utm_medium=devtools).
